src/components/UserInfo.js

```diff
@@ -1,19 +1,22 @@
 export default class UserInfo {
-  constructor(userNameSelector, userJobSelector) {
+  constructor(userNameSelector, userJobSelector, userAvatarSelector) {
     this._userName = document.querySelector(userNameSelector);
     this._userJob = document.querySelector(userJobSelector);
+    this._userAvatar = document.querySelector(userAvatarSelector);
   }
 
   getUserInfo() {
     const userInfo = {
       name: this._userName.innerText,
-      job: this._userJob.innerText,
+      about: this._userJob.innerText,
+      avatar: this._userAvatar.src,
     };
     return userInfo;
   }
 
-  setUserInfo({ name, job }) {
+  setUserInfo({ name, about, avatar }) {
     this._userName.innerText = name;
-    this._userJob.innerText = job;
+    this._userJob.innerText = about;
+    this._userAvatar.src = avatar;
```

Comment:
Needs correction: It's needed to create a separate setter for avatar `setAvatar` and use it when needed to update only the avatar, because user info and avatar are controlled by different forms.

When initializing user info you can use bit methods:

```js
userInfo.setUserInfo({ title, description })
userInfo.setAvatar(avatar)
```